source/js/primary-nav.js

```diff
@@ -20,6 +20,20 @@ let primaryNav = {
       }
     }
 
+    const summaryBar = document.querySelector(".article-navbar");
+    if(summaryBar) {
+      const heroHeight = document.querySelector('.publication-hero-container').offsetHeight;
+      const navHeight = document.querySelector(".primary-nav-container").offsetHeight;
+      const total = heroHeight + navHeight - summaryBar.offsetHeight;
```

Comment:
@mmmavis I'm not certain this is the best place for this chunk of code. Do you have a recommendation for a better location?

Reply:
I think to follow the existing file structure, let's create a file under `source/js/foundation/template-js-handler/` for this.

Reply:
Done, thank you!